plugins/module_utils/acm.py

```diff
@@ -63,7 +63,15 @@ def delete_certificate_with_backoff(self, arn):
     @AWSRetry.jittered_backoff(delay=5, catch_extra_error_codes=["RequestInProgressException"])
     def list_certificates_with_backoff(self, statuses=None):
         paginator = self.client.get_paginator("list_certificates")
-        kwargs = dict()
+        # `list_certificates` requires explicit key type filter, or it returns only RSA_2048 certificates
+        kwargs = {
+            "Includes": {
+                "keyTypes": [
+                    "RSA_1024", "RSA_2048", "RSA_3072", "RSA_4096",
+                    "EC_prime256v1", "EC_secp384r1", "EC_secp521r1",
```

Comment:
```suggestion
                    "RSA_1024",
                    "RSA_2048",
                    "RSA_3072",
                    "RSA_4096",
                    "EC_prime256v1",
                    "EC_secp384r1",
                    "EC_secp521r1",
```

With release 6 we've switched to using the black code style: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html this helps us to try and bring a more unified code style across code that's been written by a large number of contributors.